demos/storybook/stories/list-item-tag/within-an-InfoListItem.stories.ts

```diff
@@ -6,9 +6,18 @@ export const withinAnInfoListItem = (): any => ({
             <div pxb-title>Info List Item </div>
             <div pxb-subtitle>with a ListItemTag component to the right</div>
             <mat-icon pxb-icon [style.color]="colors.blue[500]">battery_charging_full</mat-icon>
-            <div pxb-right-content style="width: 160px; display: flex; justify-content: space-between;">
-                <pxb-list-item-tag label="Build Passing" [backgroundColor]="colors.green[300]" [fontColor]="colors.black[900]"></pxb-list-item-tag>
-                <pxb-list-item-tag label="5 Bugs" [backgroundColor]="colors.red[300]" [fontColor]="colors.black[900]"></pxb-list-item-tag>
+            <div pxb-right-content>
+                <pxb-list-item-tag 
+                    label="Build Passing" 
+                    [style.marginRight.px]="16"
+                    [backgroundColor]="colors.green[300]" 
+                    [fontColor]="colors.black[900]">
+                </pxb-list-item-tag>
```

Comment:
set marginLeft to 16 too for RTL

Reply:
Now I remember why I did this as a flex space-inbetween.

Reply:
Ok this is fixed now, deploy incoming.